Dockerfile

```diff
@@ -0,0 +1,31 @@
+# select build image
+FROM rust as builder
+
+# create a new empty shell project
+RUN USER=root cargo new --bin trin
```

Comment:
Is there any reason not to use `COPY . .` here instead of creating an empty project and then copying all folders one by one? We can use also `.dockerignore` [file](https://docs.docker.com/engine/reference/builder/#dockerignore-file) if we want to exclude something.